modules/utils.lua

```diff
@@ -147,6 +149,7 @@ if isServer then
         model = type(model) == 'string' and joaat(model) or (model --[[@as integer]])
         local ped = GetPlayerPed(source)
 
+        ---@diagnostic disable-next-line: redundant-return-value
```

Comment:
It might be better to *actually* avoid returning more than we claim to. It'd probably be annoying to debug in certain use cases. So just save the first value in a variable and return it, ignoring the second one.